src/plugins/imagery/components/ImageryView.vue

```diff
@@ -375,14 +375,11 @@ export default {
       return age < cutoff && !this.refreshCSS;
     },
     canTrackDuration() {
-      let hasClock;
       if (this.timeContext) {
-        hasClock = this.timeContext.clock();
+        return this.timeContext.isRealTime();
```

Comment:
We should default this.timeContext to the global time context (ie. openmct.time) and avoid this boilerplate. Indepentent time contexts and the global time context have the same interface for this reason.